airbyte-webapp/src/components/CreateConnection/SchemaError.tsx

```diff
@@ -0,0 +1,19 @@
+import { JobItem } from "components/JobItem/JobItem";
+import { Card } from "components/ui";
+
+import { LogsRequestError } from "core/request/LogsRequestError";
+import { useConnectionFormService } from "hooks/services/ConnectionForm/ConnectionFormService";
+import { SchemaError as SchemaErrorType } from "hooks/services/useSourceHook";
+
+import { TryAfterErrorBlock } from "./TryAfterErrorBlock";
+
+export const SchemaError = ({ schemaError }: { schemaError: SchemaErrorType }) => {
```

Comment:
This is used on both the connection replication tab and create connection form. Unsure if it should be moved but I don't believe there's a good central location.

Reply:
I believe it's the only one.